### PR TITLE
[Caplin Archive node]: full sepolia read experiment bug fixes

### PR DESCRIPTION
**Pull request by Giulio2002**
Bugs fixed:
* CurrentEpochAttestations misbehaviour near genesis.
* Fixed Participation indices at the Altair Fork Boundary.
* Confusing logs demoted to DBUG
* Added 6 slots of margin of error for the forward downloader
* Fixed nil exception in exits processing
* Keep 1 state in the hot storage for reorgs at all costs
* Fixed `make len: out range`

### CODE REVIEW

File: cl/antiquary/state_antiquary.go [hunk 1]
```diff
@@ -826,17 +829,17 @@ func (s *Antiquary) dumpPayload(k []byte, v []byte, c *etl.Collector, b *bytes.B
...
-// 	if err != nil {
-// 		s.logger.Error("Failed to encode full beacon state", "err", err)
-// 		return
-// 	}
-// 	// just dump it in a.txt like an idiot without afero
-// 	if err := os.WriteFile("b.txt", b, 0644); err != nil {
-// 		s.logger.Error("Failed to write full beacon state", "err", err)
-// 	}
-// }
+func (s *Antiquary) dumpFullBeaconState() {
```
Comment (Giulio2002) [inline cl/antiquary/state_antiquary.go:832]: Comment

File: cl/antiquary/state_antiquary.go [hunk 2]
```diff
@@ -434,6 +434,9 @@ func (s *Antiquary) IncrementBeaconState(ctx context.Context, to uint64) error {
 		if err := transition.TransitionState(s.currentState, block, blockRewardsCollector, fullValidation); err != nil {
 			return err
 		}
+		if s.currentState.Slot() >= 1601 && s.currentState.Slot() <= 1663 {
```
Comment (Giulio2002) [inline cl/antiquary/state_antiquary.go:437]: Comment

File: cmd/capcli/cli.go [hunk 3]
```diff
@@ -893,6 +893,14 @@ func (r *RetrieveHistoricalState) Run(ctx *Context) error {
 		return err
 	}
 	if hRoot != wRoot {
+		prevPartIdxs := haveState.PreviousEpochParticipation()
```
Comment (Giulio2002) [inline cmd/capcli/cli.go:896]: Remove

File: rpc/websocket.go [hunk 4]
```diff
@@ -58,6 +58,7 @@ func (s *Server) WebsocketHandler(allowedOrigins []string, jwtSecret []byte, com
 		if jwtSecret != nil && !CheckJwtSecret(w, r, jwtSecret) {
 			return
 		}
+
```
Comment (Giulio2002) [inline rpc/websocket.go:61]: Remove

**Review by bayram98 — APPROVED** [review]

**Pull request merged**